sql/sql_admin.h

```diff
@@ -360,17 +360,11 @@ class Sql_cmd_alter_user_default_role : public Sql_cmd {
 
 enum alter_instance_action_enum {
   ROTATE_INNODB_MASTER_KEY,
-<<<<<<< HEAD
   ROTATE_INNODB_SYSTEM_KEY,
   ROTATE_REDO_SYSTEM_KEY,
 #ifdef WITH_WSREP
   ROTATE_GCACHE_MASTER_KEY,
 #endif
-||||||| merged common ancestors
```

Comment:
ROTATE_INNODB_SYSTEM_KEY and ROTATE_REDO_SYSTEM_KEY must be removed from PXC sources as well.

Reply:
Done.